linera-sdk/src/contract/runtime.rs

```diff
@@ -262,17 +262,15 @@ where
         application: ApplicationId<A>,
         call: &A::Operation,
     ) -> A::Response {
-        let call_bytes = bcs::to_bytes(call)
-            .expect("Failed to serialize `Operation` type for a cross-application call");
```

Comment:
The message in the expect is a bit more precise.

Reply:
Yes, that is right. We lose the context that it was a "cross-application call". I think that this is not a major problem. 

Note also that the error messages that we obtain are better in two ways:
* We get the error that occurred in the serialization.
* We get the input.

Reply:
If that is viewed as a major problem, then we could pass the context in the function call.

Reply:
How about keeping all the `.expect` as they are, and use `bcs::..().map_err(String::from)` in the abi methods?

Reply:
Using the `expect` is a good idea indeed as the error show up.
Using `String::from` is not a good idea since if we want to debug serialization error, we need to have the input.